tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -485,4 +823,65 @@ func setVpxdTaskTimeout(ctx context.Context, taskTimeout int) {
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	connect(ctx, &e2eVSphere)
+
+	govmomiClient := newClient(ctx, &e2eVSphere)
+	pc = newPbmClient(ctx, govmomiClient)
+}
+
+// writeKnownData2PodInParallel writes known 1mb data to a file in given pod's volume until 100mb is left in the volume
+// in parallel
+func writeKnownData2PodInParallel(
+	f *framework.Framework, pod *v1.Pod, testdataFile string, wg *sync.WaitGroup, size ...int64) {
+
+	defer wg.Done()
+	writeKnownData2Pod(f, pod, testdataFile, size...)
+}
+
+// writeKnownData2Pod writes known 1mb data to a file in given pod's volume until 300mb is left in the volume
+func writeKnownData2Pod(f *framework.Framework, pod *v1.Pod, testdataFile string, size ...int64) {
+	_ = framework.RunKubectlOrDie(pod.Namespace, "cp", testdataFile, fmt.Sprintf(
+		"%v/%v:/mnt/volume1/testdata", pod.Namespace, pod.Name))
+	fsSize, err := getFSSizeMb(f, pod)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	iosize := fsSize - 300
+	if len(size) != 0 {
+		iosize = size[0]
+	}
+	iosize = iosize / 100 * 100 // will keep it as multiple of 100
+	framework.Logf("Total IO size: %v", iosize)
+	for i := int64(0); i < iosize; i = i + 100 {
+		seek := fmt.Sprintf("%v", i)
+		cmd := []string{"--namespace=" + pod.Namespace, "-c", pod.Spec.Containers[0].Name, "exec", pod.Name, "--",
+			"/bin/sh", "-c", "dd if=/mnt/volume1/testdata of=/mnt/volume1/f1 bs=1M count=100 seek=" + seek}
+		_ = framework.RunKubectlOrDie(pod.Namespace, cmd...)
+	}
+	cmd := []string{"--namespace=" + pod.Namespace, "-c", pod.Spec.Containers[0].Name, "exec", pod.Name, "--",
+		"/bin/sh", "-c", "rm /mnt/volume1/testdata"}
+	_ = framework.RunKubectlOrDie(pod.Namespace, cmd...)
+}
+
+// verifyKnownDataInPod verify known data on a file in given pod's volume in 100mb loop
+func verifyKnownDataInPod(f *framework.Framework, pod *v1.Pod, testdataFile string, size ...int64) {
+	fsSize, err := getFSSizeMb(f, pod)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	iosize := fsSize - 300
```

Comment:
We can take 300  as a constant as it is being used in many places

Reply:
done and changed it to 200